src/Reducers/requestInboxAlertReducers.js

```diff
@@ -0,0 +1,23 @@
+import initialState from './initialState';
+
+const requestInboxAlertReducers = (
+  state = initialState.requestInboxAlert,
+  action
+) => {
+  switch (action.type) {
+    case 'REQUEST_INBOX_ALERT': {
+      return { ...state };
+    }
+    case 'REQUEST_INBOX_ALERT_SUCCESS': {
+      const newAlerts = action.payload.filter(
+        alerts => alerts.govstatus === 'Pending'
+      ).length;
+
+      return { ...state, alert: newAlerts };
```

Comment:
I saw that you you are changing type.  Your initial state has obj your passing to number here. This makes me thing we need to use typescript